apps/user-ms/src/test/java/it.pagopa.selfcare.user/service/UserServiceTest.java

```diff
@@ -177,4 +180,28 @@ void retrieveUsersTest() {
         assertEquals(1, actual.size());
         assertEquals(userInstitution.getUserId(), actual.get(0).getUserId());
     }
+
+    @Test
+    void deleteUserInstitutionProductFound(){
+        when(userInstitutionService.deleteUserInstitutionProduct(anyString(), any(), any())).thenReturn(Uni.createFrom().item(1L));
```

Comment:
you could mock using the real value, for ex deleteUserInstitutionProduct("userId", "institutionId", "productId")